dlp/src/main/java/com/example/dlp/RiskAnalysis.java

```diff
@@ -53,90 +64,135 @@
 public class RiskAnalysis {
 
   private static void calculateNumericalStats(
-      String projectId, String datasetId, String tableId, String columnName)
+      String projectId,
+      String datasetId,
+      String tableId,
+      String columnName,
+      String topicId,
+      String subscriptionId)
       throws Exception {
     // [START dlp_numerical_stats]
-
     /**
      * Calculate numerical statistics for a column in a BigQuery table using the DLP API.
+     *
      * @param projectId The Google Cloud Platform project ID to run the API call under.
      * @param datasetId The BigQuery dataset to analyze.
      * @param tableId The BigQuery table to analyze.
      * @param columnName The name of the column to analyze, which must contain only numerical data.
+     * @param topicId The name of the Pub/Sub topic to notify once the job completes
+     * @param subscriptionId The name of the Pub/Sub subscription to use when listening for job
+     *     completion status.
      */
 
     // instantiate a client
     try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
-
-      // projectId = process.env.GCLOUD_PROJECT;
-      // datasetId = "my_dataset";
-      // tableId = "my_table";
-      // columnName = "firstName";
-
-      FieldId fieldId =
-          FieldId.newBuilder()
-              .setColumnName(columnName)
-              .build();
-
-      NumericalStatsConfig numericalStatsConfig =
-          NumericalStatsConfig.newBuilder()
-              .setField(fieldId)
-              .build();
-
       BigQueryTable bigQueryTable =
           BigQueryTable.newBuilder()
-              .setProjectId(projectId)
-              .setDatasetId(datasetId)
               .setTableId(tableId)
+              .setDatasetId(datasetId)
+              .setProjectId(projectId)
               .build();
 
+      FieldId fieldId = FieldId.newBuilder().setName(columnName).build();
+
+      NumericalStatsConfig numericalStatsConfig =
+          NumericalStatsConfig.newBuilder().setField(fieldId).build();
+
       PrivacyMetric privacyMetric =
-          PrivacyMetric.newBuilder()
-              .setNumericalStatsConfig(numericalStatsConfig)
-              .build();
+          PrivacyMetric.newBuilder().setNumericalStatsConfig(numericalStatsConfig).build();
 
-      AnalyzeDataSourceRiskRequest request =
-          AnalyzeDataSourceRiskRequest.newBuilder()
-              .setPrivacyMetric(privacyMetric)
+      String topicName = String.format("projects/%s/topics/%s", projectId, topicId);
+
+      PublishToPubSub publishToPubSub = PublishToPubSub.newBuilder().setTopic(topicName).build();
+
+      // create /action to publish job status notifications over Google Cloud Pub/Sub
```

Comment:
Nit: remove `/` in `/action`?

Apply this comment throughout this PR (if you decide to change anything).

Reply:
Done